EpicLoot/src/Magic/MagicItemEffects/ModifyAttackEitr.cs

```diff
@@ -0,0 +1,20 @@
+﻿using HarmonyLib;
+
+namespace EpicLoot.MagicItemEffects
+{
+    [HarmonyPatch(typeof(Attack), nameof(Attack.GetAttackEitr))]
+    public class ModifyAttackEitr_Attack_GetAttackEitr_Patch
+    {
+        public static void Postfix(Attack __instance, ref float __result)
+        {
+            if (__instance.m_character is Player player)
+            {
+                if (player.HasActiveMagicEffect(MagicEffectType.DoubleMagicShot, out float effectValue))
```

Comment:
Double shot should just be on the active weapon, we need to change this check to be on the weapon I think. Needs testing.

Reply:
Finished in [commit](https://github.com/OrianaVenture/Randy_Vapok_ValheimMods/pull/10/commits/5b885e9615288a1117bd17569c590ca2bd66dd9e)